.github/workflows/storybook-tests.yml

```diff
@@ -0,0 +1,17 @@
+name: Storybook Tests
+on: deployment_status
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    if: github.event.deployment_status.state == 'success'
+    steps:
+      - uses: actions/checkout@v4
+      - name: Checkout and Setup Node
+        uses: ./.github/actions/checkout-and-setup
+      - name: Install Playwright
+        run: pnpm exec playwright install --with-deps
```

Comment:
I'm curious why we need to install the Playwright browsers for the Storybook tests 🤔

Reply:
`test-runner` uses Playwright and I assume since it's visual tests they are being run in the browser. I followed the example [here](https://storybook.js.org/docs/writing-tests/test-runner#run-against-deployed-storybooks-via-github-actions-deployment).